packages/menu/src/Menu/Menu.js

```diff
@@ -60,7 +61,7 @@ function Menu({
       trapFocus={trapFocus}
       width={width}
       zIndex={1600}
-      placement='bottom-start'
+      placement={placement ?? 'bottom-start'}
```

Comment:
this didn't change the code coverage requirements? either way, could you add a test for this please.

Reply:
Added a snapshot tests since the placement is important for the popover content.

Reply:
I think this is still testing that the prop passed to the Menu controls the placement behaviour, doesn't have to be a big test but something there to make sure that we have a control and a fallback.

Reply:
ah right the fallback 😅 added the test using data-placement!